pkgs/development/tools/misc/binutils/remarkable-linker.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/bfd/elf32-arm.c b/bfd/elf32-arm.c
+index 32b94310f1..acf045841e 100644
+--- a/bfd/elf32-arm.c
++++ b/bfd/elf32-arm.c
+@@ -2260,7 +2260,7 @@ typedef unsigned short int insn16;
+ 
+ /* The name of the dynamic interpreter.  This is put in the .interp
+    section.  */
+-#define ELF_DYNAMIC_INTERPRETER     "/usr/lib/ld.so.1"
++#define ELF_DYNAMIC_INTERPRETER     "/lib/ld-linux-armhf.so.3"
```

Comment:
Do we need this? We've done elf32 dynamic linking fine without it in the past, I think.

Reply:
We override the interpreter with flags anyways, I'm pretty sure